### PR TITLE
document releasing process

### PR DESCRIPTION
**Pull request by zuntrax**
This is mostly what I did yesterday. Makes sense?

### CODE REVIEW

**Comment by simonsan** [comment]
Regarding #1133 #1134

**Comment by simonsan** [comment]
Maybe to add: I would explicitly state some info how and where to give information about the depreciation of features. I don't know if it already plays a role here or just after 1.0.0

File: doc/releasing.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+* Truncate `changelog.txt` if it exists. You might want to use this later for writing a release text.
+* Commit, PR, merge, pull as usual.
+* OPTIONAL: Set up [commit signing](https://help.github.com/en/articles/managing-commit-signature-verification), signing tags will make a "verified" badge show up on the release later.
+* Tag the merge commit, something like `git tag -s v0.5.0`. The prefixed `v` is mandatory.
+* Push the tag to GitHub.
+* Use the GitHub web interface to publish a release from the tag. Make it look nice, see previous release description. If in doubt, save the draft and have someone else proof read it.
+* In the future, kevin will automatically attach build artifacts like installers to the release. For now, you will need to do that manually. Have fun setting up a windows build environment or if you feel like being lazy, go bug someone who already has. Actually, taking care of where to build for windows before publishing the release might not be a bad idea.
+* OPTIONAL: Brag on social media. /r/openage and AoE2 Discord might be the right place to do so.
```
Comment (heinezen) [inline doc/releasing.md]: Discord --> Discords

File: doc/releasing.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+If you need help or are not completely sure what you are doing, ask in matrix and highlight `zuntrax` or someone else who has done it before. Better having someone help you than screwing up a release.
```
Comment (heinezen) [inline doc/releasing.md]: Link to the Matrix SFTtech chanel would be good :)

**Comment by simonsan** [comment]
https://pillow.readthedocs.io/en/stable/deprecations.html

As an example. I would put that as an extra point prominent into the sphinx doku.

Start could be that we changed minimal Python version to **Python 3.6** with **0.4.0**

**Comment by zuntrax** [comment]
@simonsan Not sure if I would consider updating a dependency a deprecation. I kinda see it for python2, because it's mostly a different language.

**Comment by janisozaur** [comment]
Here's a similar document I wrote on releasing OpenRCT2, perhaps you will find some inspiration in there as well: https://gist.github.com/janisozaur/693b70c5d2c583d6c6f0e7ffbc983720

**Pull request merged**